docs/specification/value-types.md

```diff
@@ -246,10 +246,11 @@ For example "one part flour two parts sugar" is acceptable.
 A composition represented as a map from the component name to the quantity.
 The quantities do not express an uncertainty, but also do not imply that there is absolute certainty in their values.
 
-Field name      | Value type | Description
-----------------|------------|------------
-`type`          | String     | "nominal\_composition"
-`quantities`    | Map[String, Number]     | Map[String, Number]
+Field name      | Value type | Default | Description
+----------------|------------|---------|-----------
+`type`          | String     |  Req.   | "nominal\_composition"
+`quantities`    | Map[String, Number]| Empty set  |   | Map[String, Number]
+`basis`         | `mass`, `volume`, `number`, `unknown` | `unknown` | The type of measurement that informed the quantity
```

Comment:
What about "other"?  I'm thinking about some of the exotic formulations use cases like [dry basis](https://en.wikipedia.org/wiki/Dry_basis).

Reply:
Dry basis is still a mass (or molar or volume) basis.  Compositions are not guaranteed to be normalized, so a user filter or Attribute Template could meet that need.

More complex expressions like molarity are so complex that explicit attributes are probably the only reasonable solution.